README.rst

```diff
@@ -0,0 +1,82 @@
+Codejail service
+===============
+
+|ci-badge| |license-badge|
+
+The repository codejailservice manages execution of untrusted code in secure sandboxes. It is designed primarily for Python execution, but
+can be used for other languages as well. The security is enforced with AppArmor.
+
+The creation of Codejail service affects the way untrusted code is executed, because it runs in a separate
+container from the one running the platform.
+
+Usage
+-----
+
+Using with `Tutor <https://docs.tutor.overhang.io>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Plugin: tutor-contrib-codejail (https://github.com/eduNEXT/tutor-contrib-codejail)
```

Comment:
Can you add it as a link? using `  ` __

Also, point that they should follow the instructions indicated in the plugin

Reply:
Done